geth/api/api.go

```diff
@@ -181,8 +181,9 @@ func (api *StatusAPI) DiscardTransactions(ids []common.QueuedTxID) map[common.Qu
 
 // JailParse creates a new jail cell context, with the given chatID as identifier.
 // New context executes provided JavaScript code, right after the initialization.
-func (api *StatusAPI) JailParse(chatID string, js string) string {
-	return api.b.jailManager.Parse(chatID, js)
+// TODO(adam): change method name.
```

Comment:
Anything holding from changing name here as well?

Reply:
In this specific place no, but I can only move this comment up to `library.go` as to get rid of it completely, we need some change from ClojureTeam. I can create a new method name and deprecate the old one, though 👍